manticore/platforms/platform.py

```diff
@@ -10,14 +12,14 @@ class OSException(Exception):
 
 
 @wrapt.decorator
-def unimplemented(wrapped, _instance, args, kwargs):
+def unimplemented(wrapped: Callable, _instance, args: Iterable, kwargs: Mapping):
```

Comment:
Did Mypy/typeguard complain about these? I would've expected to see `tuple` and `dict` as the types for these instead, but perhaps it's more general than I thought.

Reply:
Hmmm, indeed, `Iterable` and `Mapping` may indeed be more general than needed here.  I _thought_ writing `args: Tuple` and `kwargs: Dict` had been problematic, but now I'm having trouble figuring out exactly what.  `mypy` definitely doesn't complain.

Let me change this to the more specific types (`Tuple` and `Dict`).

Note, `Iterable` and `Mapping` are used for expressing certain "duck types" in Python: https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html#standard-duck-types